### DISCUSSION TITLE
OpenSSL related vulnerabilities

### DISCUSSION THREAD

**Discussion by shivpatel1** [Kong/kong#9248] [Help] [answered]
We are using Kong 2.8 docker image [Default Alpine based] which is having OpenSSL related vulnerabilities.
I need to understand how OpenSSL is used by Kong and whether the Kong is vulnerable due to those vulnerabilities ?
If Kong features are also affected then what mitigation steps we can try out and recommend ?

CVE-2022-1292
CVE-2022-2068
CVE-2022-2097

**Comment by gszr** [comment, marked as answer]
By now Kong 2.8 (and later versions) uses OpenSSL version that are not affected by the two first you link; the last one does not affect us.
